cmd/root.go

```diff
@@ -102,17 +106,11 @@ func root(cmd *cobra.Command, args []string) {
 }
 
 func printResults(results httpref.References) {
-	switch len(results) {
-	case 0:
-		fmt.Fprintf(os.Stderr, "Filter not found any results\n")
-		os.Exit(1)
-	case 1:
-		fmt.Printf("%s\n", results[0].Describe(width))
-	default:
-		for _, r := range results {
-			fmt.Printf("%s\n", r.Summarize(width))
-		}
+	if passedRenderDate.IsZero() {
+		passedRenderDate = time.Now()
+		baseRootStyle = lipgloss.NewStyle().Width(width)
```

Comment:
I'm not sure how `passedRenderDate` is being used here. How is it set? And if it shouldn't a boolean, what does using a date provide?